hack/common.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -eu
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+LOGS_ROOT="${SCRIPT_DIR}/../logs"
+MON_BENCHMARKS_ROOT="${HOME}/.rh/monitoring-benchmarks"
+CONFIG_ROOT="${SCRIPT_DIR}/../config"
+
+mkdir -p "${LOGS_ROOT}"
+export PATH="${LOCAL_ROOT}/bin:${PATH}"
+
+function date_w_format {
+  date +%Y-%m-%d--%H-%M-%S
+}
+
+function log {
+    log_file="${1}"
+    msg="${2}"
+
+    touch "${log_file}"
+    echo "[$(date_w_format)] ${msg}" | tee -a "${log_file}"
+}
+
+function cluster_config_dir {
+    cluster_name="${1}"
+
+    conf_dir="${MON_BENCHMARKS_ROOT}/clusters/${cluster_name}"
+    mkdir -p "${conf_dir}"
+    echo "${conf_dir}"
+}
+
+function create_cluster {
+    CLUSTER_NAME="${1}"
+    NUM_WORKERS="${2}"
+    OCP_RELEASE="${3}"
+
+    log_file="${LOGS_ROOT}/create_cluster-$(date_w_format).log"
+    echo "Using log file ${log_file}"
+    export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE}-x86_64"
+    log "${log_file}" "Using OCP image ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+    config_dir="$(cluster_config_dir "${CLUSTER_NAME}")"
+    log "${log_file}" "Using cluster config dir ${config_dir}"
+    
+    SSH_KEY=$(cat "${SSH_KEY_PATH}")
+    export SSH_KEY
```

Comment:
if these env vars are used only by openshift-install, I think it better to put them in that line, to avoid overriding the existing env var settings.

Reply:
that's right, I removed the exports